datacube/scripts/search_tool.py

```diff
@@ -93,6 +93,9 @@ def datasets(ctx, index, expressions) -> None:
     """
     Search available Datasets
     """
+    click.echo(
```

Comment:
Can the `deprecated=True` in click (https://click.palletsprojects.com/en/stable/commands-and-groups/#deprecating-commands) be used instead so there is a deprecation warning emitted?

Reply:
And if it's `click >= 8.2` it can also be a deprecation string. But 8.2 is quite new, so maybe we shouldn't do that.

https://click.palletsprojects.com/en/stable/api/#click.Command
`deprecated` ([bool](https://docs.python.org/3/library/functions.html#bool) | [str](https://docs.python.org/3/library/stdtypes.html#str)) – If True or non-empty string, issues a message indicating that the command is deprecated and highlights its deprecation in –help. The message can be customized by using a string as the value.